subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,17 +34,17 @@ details of 2
 
 ## Performance improvements
 
-### Faster and leaner compilation of Kotlin DSL scripts
+### Kotlin DSL script compilation is faster and consumes less memory
 
-In this release, the compilation of [Gradle Kotlin DSL](userguide/kotlin_dsl.html) scripts (`*.gradle.kts`) has been improved and is now faster and consumes less memory.
+In this release, the compilation of [Gradle Kotlin DSL](userguide/kotlin_dsl.html) scripts (`*.gradle.kts`) is now faster and consumes less memory.
```

Comment:
"in this release" and "now" is a bit repetitive. I'd replace "now faster" with "much faster".